src/flat-button.jsx

```diff
@@ -197,7 +258,7 @@ const FlatButton = React.createClass({
         onKeyboardFocus={this._handleKeyboardFocus}
         onMouseLeave={this._handleMouseLeave}
         onMouseEnter={this._handleMouseEnter}
-        onTouchStart={this._handleTouchStart}
```

Comment:
I have investigated on this. We should change those line. `_handleTouchStart` is used to detect if it's a touch event that trigger `_handleMouseEnter`